<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddObjectBasic</class>
 <widget class="QDialog" name="AddObjectBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Eradiuslbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Equatorial Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="PRadiuslbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Polar Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="MInteriaUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>620</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MInertiaUM</string>
   </property>
  </widget>
  <widget class="QLabel" name="J2lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>J2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EquatorialRadius">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Albedolbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bond Albedo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>750</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TopographicRange">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Albedo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="BBTemplbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Black Body Temp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NasaNumber">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Elipticity">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SurfaceGravity">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CoreRadius">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TopographicRangelbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tropographic Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="IrradianceUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IrradianceUM</string>
   </property>
  </widget>
  <widget class="QLabel" name="SGravlbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Surface Gravity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Irradiance">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="GMlbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GM</string>
   </property>
  </widget>
  <widget class="QLabel" name="GMUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SurfaceAcceleration">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SGravityUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SGravityUM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="GM">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Magnitude">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Maglbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Magnitude</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MomentInertia">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>620</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Quit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>750</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="NumSatlbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>680</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Satellites</string>
   </property>
  </widget>
  <widget class="QLabel" name="J2UM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>650</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>J2UM</string>
   </property>
  </widget>
  <widget class="QLabel" name="ElipticityUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ElipticityUM</string>
   </property>
  </widget>
  <widget class="QLabel" name="SAccellbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Surface Acceleration</string>
   </property>
  </widget>
  <widget class="QLabel" name="CoreRadiusUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Core RadiusUM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Volume">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Numringslbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Rings</string>
   </property>
  </widget>
  <widget class="QLabel" name="Irrlbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Irradiance</string>
   </property>
  </widget>
  <widget class="QLabel" name="EscapeVelocitylbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Escape Velocity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MedianRadius">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="VolUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="NasaNumlbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NasaNumber</string>
   </property>
  </widget>
  <widget class="QLabel" name="BBodyTempUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>590</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BBodyTempUM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MDensity">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PRadiusUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MIntertlbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Moment of Inertia</string>
   </property>
  </widget>
  <widget class="QLabel" name="MRadiusUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ERadiusUM</string>
   </property>
  </widget>
  <widget class="QLabel" name="SAccelUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SAccelUM</string>
   </property>
  </widget>
  <widget class="QLabel" name="Vollbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="CoreRadiuslbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Core Radius</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EscapeVelocity">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumberRings">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>710</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="VGAlbedolbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eliplbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Elipticity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PolarRadius">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Mass">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BlackBodyTemp">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>590</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="J2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Masslbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mass</string>
   </property>
  </widget>
  <widget class="QLabel" name="MassUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Namlbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="EVelocityUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>EVelocity</string>
   </property>
  </widget>
  <widget class="QLabel" name="ERadiusUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumberSatellites">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>680</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MDensitylbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MDensity</string>
   </property>
  </widget>
  <widget class="QLabel" name="Albedolbl_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Visual Geometric  Albedo</string>
   </property>
  </widget>
  <widget class="QLabel" name="MRadiuslbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Median Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="TopographicRangeUm">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tropographic Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="MDensityUM">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MDensityUM</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Add</sender>
   <signal>clicked()</signal>
   <receiver>AddObjectBasic</receiver>
   <slot>AddBasicData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>761</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Quit</sender>
   <signal>clicked()</signal>
   <receiver>AddObjectBasic</receiver>
   <slot>Exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>761</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddBasicData()</slot>
  <slot>Exit()</slot>
 </slots>
</ui>
